### PR TITLE
fix: incorrect code snippets, links, descriptions for create stream and batch create streams subsections

### PR DESCRIPTION
**Pull request by ryabykhms**

### CODE REVIEW

**Review by PaulRBerg — APPROVED** [review]
LGTM thanks very much for your PR @ryabykhms!

**Comment by PaulRBerg** [comment]
I will let @smol-ninja have a second review before merging

**Review by smol-ninja — APPROVED** [review]
LGTM. Thank you for the PR.

**Pull request merged**
